samples/tfm_integration/psa_level_1/README.rst

```diff
@@ -0,0 +1,145 @@
+.. _tfm_psa_level_1:
+
+TF-M PSA Level 1
+################
+
+Overview
+********
+This TF-M integration example can be used with an Arm v8-M supported board. It
+implements the RTOS vendor requirements for a PSA Certified Level 1 product.
+
+Trusted Firmware (TF-M) Platform Security Architecture (PSA) APIs
+are used for the secure processing environment, with Zephyr running in the
+non-secure processing environment.
+
+As part of the standard build process, the secure bootloader (BL2) is built, as
+well as the secure TF-M binary and non-secure Zephyr binary images. The two
+application images are then merged and signed using the public/private key pair
+stored in the secure bootloader, so that they will be accepted during the
+image validation process at startup.
+
+All TF-M dependencies will be automatically cloned in the /ext folder, and
+built if appropriate build artefacts are not detected.
+
+.. _PSA Certified Level 1:
+  https://www.psacertified.org/security-certification/psa-certified-level-1/
+
+.. note:: This example is based upon trusted-firmware-m commit 0d822496e809132e040c0a5cd556b2d5dad01ea6
+
+Building and Running
+********************
+
+This project outputs startup status and info to the console. It can be built and
+executed on an MPS2+ configured for AN521 (dual-core ARM Cortex M33).
+
+At present, this sample is only compatible with Linux and OS X based systems,
+and has been tested on the following setups:
+
+- OS X 10.14.5 natively with gcc-arm-none-eabi-7-2018-q2-update
+- Ubuntu 18.04 using Zephyr SDK 0.10.1
+
+On MPS2+ AN521:
+===============
+
+1. Build Zephyr with a non-secure configuration (``-DBOARD=mps2_an521_nonsecure``). This will also build TF-M in the ``ext/tfm`` folder if necessary.
+
+Using ``west``
+
+.. code-block:: bash
+
+   cd <ZEPHYR_ROOT>/samples/tfm_integration/psa_level_1/
+   west build -p -b mps2_an521_nonsecure ./
+
+Using ``cmake``
+
+.. code-block:: bash
+
+   cd <ZEPHYR_ROOT>/samples/tfm_integration/psa_level_1/
+   mkdir build
+   cd build
+   cmake -GNinja -DBOARD=mps2_an521_nonsecure ..
+   ninja -v
+
+2. Merge the TF-M secure and Zephyr non-secure images, and sign the final image
+
+.. code-block:: bash
```

Comment:
same here.  Indent the code block and contents (line 65-67) by an additional three spaces.